modules/shortcodes/css/jetpack-email-subscribe.css

```diff
@@ -0,0 +1,53 @@
+/**
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
```

Comment:
We generally don't include these comments in Jetpack. Only in WordPress.com's codebase.

Reply:
Removed